### ISSUE TITLE
Training- und Validierungs-/Testdaten split

### ISSUE THREAD

**Issue by redhead02** [CDL1-Sensor/Sensor_Data-Wrangling-und-EDA#13]
Frageliste die beantwortet werden muessen:

- [ ] Wie splitten wir die Datena uf? Gleichmaessig / Ungleichmaessig. (In Bezug auf Accuracy Metrik -> bevorzugt Gleichmaessige Verteilung der Klassen.
- [ ] Wie geschieht das Rolling Fenster ?
Wenn diese Linie der Datensatz waere:

&&&&&&&&&&

3 & definieren wir als 5 Sekunden. 
Nehemen wir nun jeweils immer nur 3 & oder 
nehmen wir 3 & und verschieben einen & dann einfach weiter (ueberlappung der Daten)  
-> Beim ueberlappen der Daten muessen wir auf Daten Leakage aufpassen! 
bzw. muss die Trennung schon davor geschehen!

**Comment by 7ben18** [comment]
crossvalidation sklearn evtl hilfreich: https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GroupKFold.html
